### PR TITLE
Feature/sqlsrv

### PR DESCRIPTION
**Pull request by atbreuer11**
Fix bug and add configuration file

### CODE REVIEW

File: layers/sqlsrv/Dockerfile [hunk 1]
```diff
@@ -17,14 +17,17 @@ RUN pecl install -f pdo_sqlsrv
...
-  Driver      = ODBC Driver 17 for SQL Server\n\
-  Description = My ODBC Driver 17 for SQL Server\n\
-  Trace       = No'  > /tmp/odbc.ini
-RUN echo $'[ODBC Driver 17 for SQL Server]\n\
-  Description = Microsoft ODBC Driver 17 for SQL Server\n\
-  Driver      = /opt/microsoft/msodbcsql17/lib64/libmsodbcsql-17.6.so.1.1\n\
-  UsageCount  = 1'  > /tmp/odbcinst.ini
+Description = Microsoft ODBC Driver 17 for SQL Server\n\
+Driver      = /opt/microsoft/msodbcsql17/lib64/libmsodbcsql-17.6.so.1.1\n\
+UsageCount  = 1'  > /tmp/odbcinst.ini
```
Comment (Nyholm) [inline layers/sqlsrv/Dockerfile:30]: Could you tell me why these lines are changed?

File: layers/sqlsrv/Dockerfile [hunk 2]
```diff
@@ -17,14 +17,17 @@ RUN pecl install -f pdo_sqlsrv
 RUN cp /opt/bref/lib/php/extensions/no-debug-non-zts-*/sqlsrv.so /tmp/sqlsrv.so
 RUN cp /opt/bref/lib/php/extensions/no-debug-non-zts-*/pdo_sqlsrv.so /tmp/pdo_sqlsrv.so
 
+RUN echo $'extension=/opt/bref-extra/sqlsrv.so\n\
+extension=/opt/bref-extra/pdo_sqlsrv.so' > /tmp/ext.ini
```
Comment (Nyholm) [inline layers/sqlsrv/Dockerfile:21]: Is there a problem to have both enabled?

**Review by Nyholm — COMMENTED** [review]
Make sure to enable the test too. Just remove the first `exit(0)`.

File: layers/sqlsrv/Dockerfile (hunk 1 above)
Comment (atbreuer11) [reply layers/sqlsrv/Dockerfile:30]: This change resolved the following issue for me: `SQLSTATE[IM002]: [unixODBC][Driver Manager]Data source name not found, and no default driver specified (SQL: select * from [language])`. I suppose those configuration files are rather strict.

File: layers/sqlsrv/Dockerfile (hunk 2 above)
Comment (atbreuer11) [reply layers/sqlsrv/Dockerfile:21]: Should be fine. I have had them both enabled in the past with no issues.

**Comment by atbreuer11** [comment]
Removed `exit(0)` and added a test for the PDO sqlsrv extension

File: layers/sqlsrv/Dockerfile (hunk 2 above)
Comment (Nyholm) [reply layers/sqlsrv/Dockerfile:21]: Thank you

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**